packages/dds/tree/src/test/shared-tree/fuzz/targetedFuzz.spec.ts

```diff
@@ -78,7 +91,22 @@ describe("Fuzz - Targeted", () => {
 	const random = makeRandom(0);
 	const runsPerBatch = 20;
 	const opsPerRun = 20;
+	const editGeneratorOpWeights: EditGeneratorOpWeights = {
+		insert: 5,
+		delete: 1,
+		setPayload: 1,
+		start: 3,
+		commit: 0,
+		abort: 1,
```

Comment:
I would just put these three to 0 and manually call start and abort on the tree in `performFuzzActionsAbort`.
Otherwise we run the risk of running some of the edits outside of the transaction that is getting aborted.

Reply:
removed transaction ops and manually call `transaction.start()` and `transaction.abort()` in `performFuzzActionsAbort`